server/datastore/mysql/errors.go

```diff
@@ -59,6 +60,9 @@ type existsError struct {
 }
 
 func alreadyExists(kind string, identifier interface{}) error {
+	if s, ok := identifier.(string); ok {
+		identifier = strconv.Quote(s)
```

Comment:
What drove you to add this?

Reply:
Mostly, for the message to be more clear in case `identifier` is an empty string: `something failed:  other message ...` vs `something failed: "" other message`.